### ISSUE TITLE
ofxTuio not working

### ISSUE THREAD

**Issue by wanwannaw** [memo/ofxMSAFluid#4]
hi the current version of ofxTuio (https://github.com/mkalten/ofxTuioWrapper) seems not to be working can you upload a working version of ofxTuio?

**Comment by wenbo001** [comment]
Yes, it is not work with ofxTuio (https://github.com/mkalten/ofxTuioWrapper), still miss the ofxTuio.h
